packages/git-diff/lib/helpers.js

```diff
@@ -1,11 +1,10 @@
-exports.repositoryForPath = function(goalPath) {
-  const directories = atom.project.getDirectories();
-  const repositories = atom.project.getRepositories();
-  for (let i = 0; i < directories.length; i++) {
-    const directory = directories[i];
-    if (goalPath === directory.getPath() || directory.contains(goalPath)) {
-      return repositories[i];
-    }
+const { Directory } = require('atom');
+
+const repositoryForPath = async path => {
```

Comment:
This doesn't need to be a heap defined lambda

```suggestion
async function repositoryForPath(path) {
```